webpack-test.config.js

```diff
@@ -31,6 +32,20 @@ const webpackPostLoaders = [
   }
 ];
 
+const webpackLoaders = webpackConfig.module.loaders
+// copy loaders as they are and exclude nullable ones
+  .slice( 0, webpackConfigNullableLoaders.length )
```

Comment:
I'm afraid of this, it make config more complicated and dependent, we should take a look at it from big picture and reevaluate Webpack config setup.